extensions/mssql/package.json

```diff
@@ -52,7 +52,7 @@
         "command": "mssqlCluster.task.newNotebook",
         "title": "%notebook.command.new%",
         "icon": {
-          "dark": "resources/dark/new_notebook_inverse.svg",
+          "dark": "resources/dark/new_notebook.svg",
```

Comment:
intentional? same icon for both dark and light?

Reply:
Yes Raden checked with the visual designers and confirmed that it would be the same icons for both dark and light themes.

Reply:
if that is the case, we should clean up the icons.

Reply:
yes I removed the old icons

Reply:
Please do test this with dark...